wagtail_wordpress_import/prefilters/handle_shortcodes.py

```diff
@@ -96,7 +96,9 @@ def element_name(self):
 
     @property
     def is_top_level_html_tag(self):
-        return self.is_top_level_html_tag
+        if not type(self.top_level_html_tag) == bool:
+            return True
+        return self.top_level_html_tag
```

Comment:
**Comment**: I don't understand the way this was originally written. The following class:

```python
class MyClass:
    is_foo = True

    @property
    def is_foo(self):
        return self.is_foo
```

gives the following REPL session:

```python
>>> inst = MyClass()
>>> inst.is_foo
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/nick/projects/wxd/wagtail-wordpress-import/thing.py", line 6, in is_foo
    return self.is_foo
  File "/home/nick/projects/wxd/wagtail-wordpress-import/thing.py", line 6, in is_foo
    return self.is_foo
  File "/home/nick/projects/wxd/wagtail-wordpress-import/thing.py", line 6, in is_foo
    return self.is_foo
  [Previous line repeated 996 more times]
RecursionError: maximum recursion depth exceeded
>>> 
```

Why do we need to test that this is a boolean type? Why do we need a method to access the attribute?